# HG changeset patch
# User Markus Mützel <markus.muetzel@gmx.de>
# Date 1522689301 -7200
#      Mon Apr 02 19:15:01 2018 +0200
# Branch stable
# Node ID 1e0d3ad041de03ca8f553269132dea5d06ca636e
# Parent  34a5a420208eb9006d9b416bacf8d63b9b9fffbe
Add some strings lost in update of translation files (bug #53526).

* de_DE.ts, ja_JP.ts: Re-add some translated strings that where lost when
updating the source strings.

diff --git a/libgui/languages/de_DE.ts b/libgui/languages/de_DE.ts
--- a/libgui/languages/de_DE.ts
+++ b/libgui/languages/de_DE.ts
@@ -430,17 +430,20 @@
         <source>Octave Documentation</source>
         <translation>Octave Dokumentation</translation>
     </message>
     <message>
         <location line="-220"/>
         <source>Could not copy help collection to temporary
 file. Search capabilities may be affected.
 %1</source>
-        <translation type="unfinished"></translation>
+        <translation>Hilfe-Sammlung konnte nicht in temporäre
+Datei kopiert werden. Suchfunktionen
+könnten beeinträchtigt sein.
+%1</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>Could not setup the data required for the
 documentation viewer. Only help texts in
 the Console Widget will be available.</source>
         <translation>Die für den Dokumentationsbetrachter
 benötigten Dateien konnten nicht vorbereitet werden.
@@ -465,17 +468,17 @@ Das Betrachten der Hilfe ist auf das Befehlsfenster beschränkt.</translation>
     <message>
         <location line="-22"/>
         <source>Index</source>
         <translation>Index</translation>
     </message>
     <message>
         <location line="+156"/>
         <source>Unable to register help file %1.</source>
-        <translation type="unfinished"></translation>
+        <translation>Hilfe-Datei %1 konnte nicht registriert werden.</translation>
     </message>
 </context>
 <context>
     <name>octave::documentation_dock_widget</name>
     <message>
         <location filename="../src/documentation-dock-widget.cc" line="+39"/>
         <source>Documentation</source>
         <translation>Dokumentation</translation>
@@ -1105,17 +1108,21 @@ Wählen Sie eine andere Kodierung oder brechen Sie das Speichern ab!</translation>
     </message>
     <message>
         <location line="+11"/>
         <source>The current editor contents can not be encoded
 with the selected encoding %1.
 Using it will result in data loss!
 
 Do you want to choose another encoding?</source>
-        <translation type="unfinished"></translation>
+        <translation>Der Text kann mit der gewählten
+Zeichenkodierung %1 nicht ohne Datenverlust
+gespeichert werden!
+
+Andere Zeichenkodierung wählen?</translation>
     </message>
     <message>
         <location line="+85"/>
         <source>It seems that &apos;%1&apos; has been modified by another application. Do you want to reload it?</source>
         <translation>Die Datei &apos;%1&apos; wurde von einer anderen Anwendung verändert. Soll der neue Inhalt geladen werden?</translation>
     </message>
     <message>
         <location line="+29"/>
diff --git a/libgui/languages/ja_JP.ts b/libgui/languages/ja_JP.ts
--- a/libgui/languages/ja_JP.ts
+++ b/libgui/languages/ja_JP.ts
@@ -1403,17 +1403,17 @@ for write: %2.</source>
     <message>
         <location line="+5"/>
         <source>Indent Code</source>
         <translation>コードをインデント</translation>
     </message>
     <message>
         <location line="+7"/>
         <source>Convert Line Endings to &amp;Windows (CRLF)</source>
-        <translation type="unfinished"></translation>
+        <translation>行末文字を Windows 仕様にする (CRLF) (&amp;W)</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Convert Line Endings to &amp;Unix (LF)</source>
         <translation>行末文字を Unix 仕様にする (LF) (&amp;U)</translation>
     </message>
     <message>
         <location line="+5"/>
@@ -1740,26 +1740,29 @@ Do you want to choose another name?</sou
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+20"/>
         <source>The current encoding %1
 can not be applied.
 
 Please select another one or cancel saving!</source>
-        <translation type="unfinished"></translation>
+        <translation>現在のコーデック %1
+は適用できません．別のもの選択するが保存をキャンセルしてください．</translation>
     </message>
     <message>
         <location line="+11"/>
         <source>The current editor contents can not be encoded
 with the selected encoding %1.
 Using it will result in data loss!
 
 Do you want to choose another encoding?</source>
-        <translation type="unfinished"></translation>
+        <translation>現在エディタ上のコンテンツを選択された
+コーデック %1 でエンコードできません．
+それを使用するとデータを消失します！</translation>
     </message>
     <message>
         <location line="+85"/>
         <source>It seems that &apos;%1&apos; has been modified by another application. Do you want to reload it?</source>
         <translation>&apos;%1&apos; は他のアプリケーションで変更されたいます．再読込しますか？</translation>
     </message>
     <message>
         <location line="+29"/>
@@ -3347,17 +3350,17 @@ Do you want to use the shortcut anyhow r
     <message>
         <location line="+2"/>
         <source>Move Tab Left</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+2"/>
         <source>Move Tab Right</source>
-        <translation> type=&quot;unfinished&quot;</translation>
+        <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Edit Function</source>
         <translation>関数の編集</translation>
     </message>
     <message>
         <location line="+2"/>
@@ -4261,17 +4264,17 @@ Would you like to save the current short
     <message>
         <location line="-19"/>
         <source>Show tool bar</source>
         <translation>ツールバーの表示</translation>
     </message>
     <message>
         <location line="+450"/>
         <source>Wrap long lines at current window border</source>
-        <translation type="unfinished"></translation>
+        <translation>現在のウィンドウの境界で長い行をラップ</translation>
     </message>
     <message>
         <location line="+15"/>
         <source>Indentation</source>
         <translation>インデント</translation>
     </message>
     <message>
         <location line="+40"/>
@@ -4704,17 +4707,17 @@ Would you like to save the current short
     <message>
         <location line="-290"/>
         <source>HttpProxy</source>
         <translation>HTTP プロキシ</translation>
     </message>
     <message>
         <location line="-2179"/>
         <source>Icon set for dock widgets</source>
-        <translation type="unfinished"></translation>
+        <translation>Dock widgets のためのアイコンセット</translation>
     </message>
     <message>
         <location line="-220"/>
         <source>Language (requires restart)</source>
         <translation>言語 (要再起動)</translation>
     </message>
     <message>
         <location line="-17"/>
